docs/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement CC list builder in Bugzilla backwards sync (OSIDB-386)
 - Implement validation for affects with exceptional combination of affectedness and resolution (OSIDB-361)
 - Implement validation for affects marked as WONTFIX or NOTAFFECTED with open trackers (OSIDB-364)
+- Update testrunner base image from fedora:34 to ubi8/ubi:8.7
```

Comment:
I don't think it's necessary to have a changelog entry for this change, as it's simply a technical change, just add the technical label